user-guide/Advanced_Functionality/Databases/Elasticsearch_database/Manually_Connecting_DMA_to_Elasticsearch_Cluster.md

```diff
@@ -0,0 +1,36 @@
+---
+uid: Manually_Connecting_DMA_to_Elasticsearch_Cluster
+---
+# Manually connecting a DMA to an Elasticsearch cluster
+
+You can connect a DataMiner Agent to an existing Elasticsearch cluster by manually adding a *DataBase* tag to the *DB.xml* file.
+
+> [!IMPORTANT]
+> Make sure the DataMiner Agent has an available connection to each node of the Elasticsearch cluster. To verify this, enter `http://elasticnodeip:9200/` in your browser's address bar to access the general server information.
```

Comment:
I have reworded this section slightly. Go ahead and tweak this if I accidentally adjusted the content of this step. 
+ Maybe we could add a screenshot of the general server information so that the user knows what an available connection looks like.